src/Pyz/Yves/ShopUi/Theme/default/components/molecules/date-time-picker/date-time-picker.ts

```diff
@@ -1,115 +1,71 @@
 import Component from 'ShopUi/models/component';
-import $ from 'jquery/dist/jquery';
-import 'jquery-datetimepicker/build/jquery.datetimepicker.full';
 
 export default class DateTimePicker extends Component {
     protected trigger: HTMLInputElement;
     protected dateFrom: HTMLInputElement;
     protected dateTo: HTMLInputElement;
+    protected datePicker: HTMLInputElement;
 
     protected readyCallback(): void {}
 
     protected init(): void {
-        this.trigger = <HTMLInputElement>this.getElementsByTagName('input')[0];
-        this.dateFrom = <HTMLInputElement>document.getElementById(this.dateFromId);
-        this.dateTo = <HTMLInputElement>document.getElementById(this.dateToId);
+        this.trigger = this.querySelector<HTMLInputElement>('input[type="text"]');
+        this.dateFrom = document.getElementById(this.dateFromId) as HTMLInputElement;
+        this.dateTo = document.getElementById(this.dateToId) as HTMLInputElement;
+        this.datePicker = this.querySelector<HTMLInputElement>(`.${this.jsName}__datepicker`);
+        this.datePicker.value = this.trigger.value;
```

Comment:
```suggestion
        this.datePicker.value = this.trigger.value;
```

is it part of `datePickerInit`?